microsoft-edge/devtools-guide-chromium/whats-new/2025/02/devtools-133.md

```diff
@@ -0,0 +1,92 @@
+---
+title: What's New in DevTools (Microsoft Edge 133)
+description: "Move a tab left or right within the Elements or Sources tool. Screen reader improvements. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 2/06/2025
+---
+# What's New in DevTools (Microsoft Edge 133)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Move a tab left or right within the Elements or Sources tool
+
+<!-- Subtitle: Move a tab left or right within a tab group in the Elements or Sources tool by right-clicking the tab. -->
+
+Previously, within the **Elements** tool and **Sources** tool, you could rearrange the tabs within a tab group only by dragging them.  This can be difficult when using a trackball or head pointer.  Now, you can also move a tab by right-clicking the tab and then selecting **Move left** or **Move right**.
```

Comment:
@leahmsft should we also mention that this is an improvement for keyboard users? I'm assuming these new context menu items are keyboard accessible.

Reply:
result: 
```
This can be difficult when using a trackball, head pointer, or keyboard.  Now, you can also move a tab by right-clicking the tab (or pressing the **Context menu** key) and then selecting **Move left** or **Move right**.
```
https://www.howtogeek.com/425623/what-is-the-menu-key-for-and-how-to-remap-it/